installer/manifests/keptn/charts/control-plane/templates/core.yaml

```
@@ -201,6 +201,8 @@ spec:
               value: {{ .Values.mongo.auth.bridgeAuthDatabase | default "openid" }}
             - name: CONFIG_DIR
               value: "/config"
+            - name: AUTOMATIC_PROVISIONING_MSG
+              value: {{.Values.bridge.automaticProvisioningMsg}}
```

```suggestion
              value: {{. Values.bridge.automaticProvisioningMsg }}
```

---

@mowies I think you meant:

```suggestion
              value: {{ .Values.bridge.automaticProvisioningMsg }}
```

---

woops, yes exactly 😄